code/game/objects/structures/watercloset.dm

```diff
@@ -445,6 +567,45 @@
 
 	if(!(istype(O)))
 		return
+	if(iswrench(O) && !istype(src, /obj/structure/sink/puddle))		//puddles laugh at your attempt to disconnect them. you can't remove a puddle from the ground with a wrench!
+		var/obj/item/weapon/wrench/W = O
+		if(anchored)
+			var/response = alert(user, "Would you like to wash [W] or disconnect [src]?", "Wrench", "Wash", "Disconnect")
+			if(response == "Disconnect")
+				user.visible_message("[user] is disconnecting [src].", "<span class='notice'>You are disconnecting [src]...</span>")
+				if(do_after(user, 40 * W.toolspeed, target = src))
+					if(!loc || !anchored)
+						return
+					user.visible_message("[user] disconnected [src]!", "<span class='notice'>You disconnected [src]!</span>")
```

Comment:
`"<span class='notice'>[user] disconnects [src]!</span>", "<span class='notice'>You disconnect [src]!</span>"`